### DISCUSSION TITLE
Accessibility for admin panel

### DISCUSSION THREAD

**Discussion by robwent** [strapi/strapi#6465] [Legacy Topics]
Hi,

I was wondering if it's possible to override admin styling without changes being wiped on an update.

For me, the colours of the menu are very hard to read as the title and item colours are very close to the background colour.

![strapi-menu](https://user-images.githubusercontent.com/411678/83341401-fa0dcc80-a2da-11ea-90eb-c79799c501f7.PNG)

Everything in the main panel seems fine.

**Comment by robwent** [comment]
I tried following this guide: https://strapi.io/documentation/v3.x/admin-panel/customization.html#development-mode

Now I have LeftMenuLink in admin/src/components but changing the values in A.js doesn't seem to make any changes after restarting.

Also, I just tried hardcoding #fff, but where do the ${props} come from?
I saw the docs about the .env file, but how would you define something like

    ${props => props.theme.main.colors.leftMenu['link-color']}

in that file?

**Reply by sunnysonx** [reply]
> but where do the ${props} come from?
> ${props => props.theme.main.colors.leftMenu['link-color']}

`props` always come the component who is calling/importing LeftMenuLink.

In your case you need `props.theme.main.colors`, you can find their values in `.cache\admin\src\themes\colors.js`, duplicate that file into `admin\src\themes\colors.js` and edit colors as you want.

**Comment by robwent** [comment]
Is this in the right place? It doesn't seem to change the output after restarting.

![strapi](https://user-images.githubusercontent.com/411678/83452665-09168b00-a451-11ea-886a-e2761bef2bc8.PNG)
